app/Main.hs

```diff
@@ -60,60 +60,58 @@ formatOne ::
   Mode ->
   -- | Configuration
   Config RegionIndices ->
-  -- | File to format or stdin as 'Nothing'
+  -- | File path supplied on the command line
```

Comment:
I think the old comment was better because it explained the meaning of the argument when it is `Nothing`.

Reply:
Yes, I agree on this. I changed it because of a misunderstanding so, I will revert the change :)